src/python/WMCore/DataStructs/Mask.py

```diff
@@ -139,7 +140,7 @@ def addRunAndLumis(self, run, lumis=None):
         if not isinstance(lumis, list):
             lumis = list(lumis)
 
-        if not run in self['runAndLumis'].keys():
+        if not run in self['runAndLumis']:
```

Comment:
For future PRs, I'd suggest the construction something not in, e.g.:
```
if run not in self['runAndLumis']:
```

No need to update this PR only for these 2 cases though (including the one below)

Reply:
you are correct, I promise I have explicitely added this to the list of checks before requesting a review. :sweat_smile: 
I made these changes.